mantidproject/mantid
# Workbench: Slice Viewer scale updates when changes are made via figure options

**Description of work.**
In Workbench, changing the colormap or the min/max value using the figure options dialog now updates the colorbar scale as well as the figure. Because of the way the options dialog works the changes to the colorbar only take effect when the dialog is closed, rather than when Accept is clicked.

**To test:**
1. In Workbench, load a workspace, right-click it and select Show Slice Viewer.
2. Click the cog icon to open figure options, go to the Images tab, and change the Colormap, Min Value, and Max Value.
3. Click OK and check that the colorbar on the right of the figure changes correctly. Also ensure that the labels for the colormap, min, and max value have updated.

Fixes #26600 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
